### ISSUE TITLE
Problem in SDF LOSS

### ISSUE THREAD

**Issue by ocissor** [JiangWenPL/multiperson#44]
Hi, I am trying to use your SDF loss function implementation for my project. I am facing an issue described below

#CODE SNIPPET
import sdf
sdf_loss = sdf.sdf_loss.SDFLoss(np.array(human_parameters['faces'].detach().cpu()))
translations = torch.ones((num_people,3), device = 'cuda:0')
vertices = params['pred_vertices']
sdf_loss(vertices,translations) 

There are people in the image and I have generated the vertices and faces of those people. I want to calculate the sdf loss function value when all have identical translations ( that is same tx, ty, and tz) and this loss is coming out to be zero I don't understand why because if all the people have the same translation values there is overlap and that implies there are collisions but why is the sdf loss value zero?

Thank you!
